### PR TITLE
Reserve Factor Updates Feb 29

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Update reserve factors of 4 pools.
Add tests.

### CODE REVIEW

**Review by sakulstra — DISMISSED** [review]

File: src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Reserve Factor Updates (February 29, 2024)"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/18"
+snapshot: "No snapshot for Direct-to-AIP"
```
Comment (sakulstra) [inline src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md:5]:
```suggestion
```

File: src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+| DAI   | 81.00%         |
+| USDC  | 83.00%         |
+| USDT  | 82.00%         |
+| MATIC | 99.99%         |
+
+## References
+
+- Implementation: [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/AaveV2Polygon_ReserveFactorUpdatesFebruary292024_20240229.sol)
+- Tests: [AaveV2Polygon](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/AaveV2Polygon_ReserveFactorUpdatesFebruary292024_20240229.t.sol)
+- [Snapshot](No snapshot for Direct-to-AIP)
```
Comment (sakulstra) [inline src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md:35]:
```suggestion
```

File: src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Reserve Factor Updates (February 29, 2024)"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937/18"
+snapshot: "No snapshot for Direct-to-AIP"
+---
+
+## Simple Summary
+
+This AIP is a continuation of continuation of proposal 30 on Governance v3 and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
```
Comment (sakulstra) [inline src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md:10]:
```suggestion
This AIP is a continuation of proposal 30 on Governance v3 and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
```

File: src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Motivation
+
+This AIP will reduce deposit yield for assets on Polygon v2 by increasing the RF. With this upgrade being passed, users will be further encouraged to migrate from Polygon v2 to v3.
+
+Increasing the RF routes a larger portion of the interest paid by users to Aave DAO's Treasury. User's funds are not at risk of liquidation and the borrowing rate remains unchanged.
+
+Of the assets with an RF set at 99.99%, there is no change. All other asset reserves will have the RF increased by 5%.
```
Comment (sakulstra) [inline src/20240229_AaveV2Polygon_ReserveFactorUpdatesFebruary292024/ReserveFactorUpdatesFebruary292024.md:18]:
```suggestion
Of the assets with an RF set at 99.99%, there is no change. All other asset reserves will have the RF increased by up to 5%.
```

**Comment by efecarranza** [comment]
@sakulstra thank you, updated

**Review by rustboyar — APPROVED** [review]

**Pull request merged**